packages/python/plotly/plotly/express/_imshow.py

```diff
@@ -1,6 +1,11 @@
 import plotly.graph_objs as go
 from _plotly_utils.basevalidators import ColorscaleValidator
 from ._core import apply_default_cascade
+from io import BytesIO
+import base64
+from .imshow_utils import rescale_intensity, _integer_ranges, _integer_types
+import pandas as pd
+import png
```

Comment:
are we making `png` a top-level dependency for the whole library?

Reply:
that's what I was thinking since it's a small Python-only lib and optional deps are a pain to handle.

Reply:
and it has no other dependencies than the standard library

Reply:
I like inlining, per conversation this morning :)